commands/types/set/set.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This file is not needed anymore.

Reply:
Fixed: [a3c9d21](https://github.com/hazelcast/hazelcast-commandline-client/pull/45/commits/a3c9d213edcf6740fd480352948d34aaa5e57b3d).